indigo_app/templates/indigo_app/tasks/my_tasks.html

```diff
@@ -0,0 +1,94 @@
+{% extends "main.html" %}
+
+{% block content %}
+<div class="container mt-3">
+  <div class="row">
+    <div class="col-md-6">
+      <div class="card task-list-card-full-height task-list-card-radius">
+        <h6 class="text-center my-3">Your current tasks</h6>
+
+        <div class="task-list-card-scroll">
+          {% for task in open_assigned_tasks %}
+          <div class="task-list-card my-2 mx-3 p-2 task-list-card-radius task-list-card-current-task">
+            <div class="text-muted">#{{ task.pk }}</div>
+            <a href="{% url 'task_detail' place=task.place.place_code pk=task.pk %}">{{ task.title }}</a>
+
+            {% if task.document %}
+            <div>
+              <a href="{% url 'document' doc_id=task.document.id %}" class="text-muted">{{ task.document.title }} @ {{ task.document.expression_date|date:'Y-m-d' }} · {{ task.document.language }}</a><br>
+              <span class="text-muted">{{ task.work.frbr_uri }}</span>
+            </div>
+            {% elif task.work %}
+            <div>
+              <a href="{% url 'work' frbr_uri=task.work.frbr_uri %}" class="text-muted" data-popup-url="{% url 'work_popup' frbr_uri=task.work.frbr_uri %}">{{ task.work.title }}</a><br>
+              <span class="text-muted">{{ task.work.frbr_uri }}</span>
+            </div>
+            {% endif %}
+
+          </div>
+          {% endfor %}
+        </div>
+      </div>
+    </div>
+
+    <div class="col-md-3">
+      <div class="card task-list-card-full-height task-list-card-radius">
+        <h6 class="text-center my-3">Tasks waiting for approval</h6>
+
+        <div class="task-list-card-scroll">
+          {% for task in tasks_pending_approval %}
+          <div class="task-list-card my-2 mx-3 p-2 task-list-card-radius task-list-card-approval-task">
+            <div class="approval-heading-span">
+              <div class="text-muted">#{{ task.pk }}</div><i class="fas fa-fw task-icon-{{ task.state }} text-muted"></i>
```

Comment:
I think we should be consistent with iconography and use the same task indicators everywhere. I know right now the icons may not align with the new task design, but let's include them and then evolve them everywhere. They may look a bit better wrapped in `<div class="small">`?

Reply:
Done, I've added the icons for all three columns.